lib.js

```diff
@@ -25,6 +25,62 @@ function withOptions (options) {
   var db = new PouchDB(options.database)
   var configurationId = 'sense-dispatch-configuration'
   var client = new raven.Client(sentryEndpoint)
+  // change feeds timeout, and Pouch will not store the last sequence
+  // number for us. The reactive logic will restart the change feed,
+  // but in that short time interval we might lose some events if we
+  // just used "now" as a parameter for `since`. Thus we need to keep
+  // track of the last sequence number for every change we follow. See
+  // also #12
+  function followerFactory (name, options) {
+    var since = 'now'
+    // sometimes we might not receive any `change` event, thus in
+    // order to get a recent value for `since` in any case, we create
+    // a changes feed which has `live` not set, and we intercept the
+    // `complete` event
+    var oneShotOptions = _.defaults(options, { since: 'now' })
+    db.changes(oneShotOptions).on('complete', function (info) {
+      since = info.last_seq
+      log.debug(name + ' feed stored sequence number ' + since)
+    })
+    return function () {
+      // some options are always used
+      var complete = _.defaults(options, {
+        live: true,
+        include_docs: true,
+        since: since
```

Comment:
Ok thanks, but, if you want `since=now` on the first creation of the the changes follower, why dont you attach the `complete` event handler to the follower which also tracks `change` and `error` and set `since` there. Then you would not create a seperate changes stream handler just for setting since there. Maybe im not completely understanding this yet.